tests/integrations/python/tests/test_anthropic.py

```
@@ -1773,6 +1779,1037 @@ def test_32_document_text_input(self, anthropic_client, test_config, provider, m
+                            print(f"✓ Found result: {result.url}")
+        
+        # Validate domain filtering
+        from .utils.common import validate_domain_filter
+        if len(search_results) > 0:
+            validate_domain_filter(search_results, blocked=["reddit.com", "twitter.com", "x.com"])
+            print(f"✓ All {len(search_results)} results respect blocked_domains filter")
+        
+        print(f"✓ Web search with blocked_domains test passed!")
+
+    @pytest.mark.parametrize("provider,model", get_cross_provider_params_for_scenario("web_search"))
+    def test_42_web_search_multi_turn(self, anthropic_client, test_config, provider, model):
```

Ruff (0.14.11)

Unused method argument: `test_config`

(ARG002)